frontend/src/js/components/devices/dialogs/McuDeviceOnboarding.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { Typography } from '@mui/material';
+
+import { getOrganization } from '@northern.tech/store/selectors';
+import { setOnboardingApproach } from '@northern.tech/store/thunks';
+
```

Comment:
```suggestion
import { useAppDispatch } from '@northern.tech/store/store';

```
and in the other files probably...